epam-assets/theme/theme_electric.scss

```diff
@@ -541,43 +593,117 @@
+                --uui-tag-caption: var(--uui-text-primary);
+                --uui-tag-bg-hover: var(--uui-neutral-10);
+                --uui-tag-border: var(--uui-neutral-30);
+                --uui-tag-border-hover: var(--uui-neutral-30);
+
+                .uui-count_indicator {
+                    --uui-count_indicator-bg: var(--uui-neutral-30);
+                    --uui-count_indicator-border: var(--uui-neutral-30);
+                }
             }
         }
 
-        &.uui-color-neutral {
-            --uui-tag-bg: var(--uui-neutral-30);
-            --uui-tag-bg-hover: var(--uui-neutral-40);
-            --uui-tag-border: var(--uui-neutral-30);
-            --uui-tag-border-hover: var(--uui-neutral-40);
-            --uui-tag-caption: var(--uui-text-primary);
-            --uui-tag-fill: var(--uui-neutral-60);
-
-            .uui-count_indicator {
-                --uui-count_indicator-bg: var(--uui-neutral-0);
-                --uui-count_indicator-border: var(--uui-neutral-0);
+        &.uui-color-night700 {
+            &.uui-fill-solid {
+                .uui-count_indicator {
+                    --uui-count_indicator-border: color-mix(in srgb, var(--uui-neutral-0) 50%, transparent);
+                    --uui-count_indicator-bg: color-mix(in srgb, var(--uui-neutral-0) 10%, transparent);
+                }
+            }
+
+            &.uui-fill-outline {
+                --uui-tag-caption: var(--uui-text-primary);
+                --uui-tag-bg: var(--uui-neutral-20);
+                --uui-tag-bg-hover: var(--uui-neutral-30);
+                --uui-tag-border: var(--uui-neutral-50);
+                --uui-tag-border-hover: var(--uui-neutral-50);
             }
         }
 
-        &.uui-fill-outline {
-            &.uui-color-warning, &.uui-color-sun {
+        &.uui-color-warning, &.uui-color-sun {
+            --uui-tag-fill: var(--uui-neutral-100);
+
+            &.uui-fill-solid {
+                --uui-tag-caption: var(--uui-neutral-80);
+
+                .uui-count_indicator {
+                    --uui-count_indicator-bg: color-mix(in srgb, var(--uui-neutral-90) 5%, transparent);
+                    --uui-count_indicator-border: color-mix(in srgb, var(--uui-neutral-90) 30%, transparent);
+                    --uui-count_indicator-caption: var(--uui-neutral-80);
+                }
+            }
+
+            &.uui-fill-outline {
+                --uui-tag-caption: var(--uui-neutral-95);
+
                 .uui-count_indicator {
                     --uui-count_indicator-bg: var(--uui-neutral-0);
                     --uui-count_indicator-border: var(--uui-neutral-40);
                 }
             }
+        }
 
-            &.uui-color-neutral {
-                --uui-tag-bg: var(--uui-neutral-20);
-                --uui-tag-bg-hover: var(--uui-neutral-30);
-                --uui-tag-border: var(--uui-neutral-40);
+        &.uui-color-neutral, &.uui-color-night300 {
```

Comment:
we don't need to have alias 'night300' for the 'neutral' color

Reply:
fixed